external/os-guides/centos/centos-minimal.md

```diff
@@ -23,21 +23,23 @@ Customization
 
 ### Clone
 
-It is highly recommended not to make any changes in the original template, but use a **clone** instead. The following command clones the template. (Replace your-new-clone with your desired name.)
+It is highly recommended not to make any changes in the original template, but use a **clone** instead. The following command clones the template. (Replace `your-new-clone` with your desired name.)
 
 ~~~
 [user@dom0 ~]$ qvm-clone centos-7-minimal your-new-clone
 ~~~
 
 You must start the template in order to customize it.
 
-**Customizing** the template for specific use cases normally requires installing additional packages. In the running template, use yum to install the required packages as follows(replace packages with a space-delimited list of packages to be installed):
+**Customizing** the template for specific use cases normally requires installing additional packages. In the running template, use yum to install the required packages as follows (replace packages with a space-delimited list of packages to be installed):
 
 ~~~
 [user@your-new-clone ~]$ sudo yum install packages
 ~~~
 
-### Passwordless root
+### Qubes R4.0
```

Comment:
According to the guidelines, we should write version-specific code like this.
https://www.qubes-os.org/doc/doc-guidelines/#version-specific-documentation